src/cds_objects.h

```diff
@@ -36,7 +36,6 @@
 #include <map>
 #include <memory>
 #include <vector>
-namespace fs = std::filesystem;
```

Comment:
If the alias is not needed, also the include must be in one of the others. Did you try to remove it as well?